uitools/cpp/Esri/ArcGISRuntime/Toolkit/FloorFilterController.cpp

```diff
@@ -829,22 +828,31 @@ namespace Toolkit {
     EnvelopeBuilder b{envelope};
     b.expandByFactor(ZOOM_PADDING);
 
+    auto c = std::make_shared<QMetaObject::Connection>();
```

Comment:
🙃  Minor tweak: it's better to have two `c` declarations as opposed to one. Overall, keep declarations in the tightest scope possible.